elements-code-tutorial/advanced-examples.md

```diff
@@ -839,8 +839,7 @@ TOKEN_ADDR=$NEWADDR
 
 CONTRACT='{"entity":{"domain":"'$DOMAIN'"},"issuer_pubkey":"'$PUBKEY'","name":"'$NAME'","precision":'$PRECISION',"ticker":"'$TICKER'","version":'$VERSION'}'
 
-# We will hash using openssl, other options are available
-CONTRACT_HASH=$(echo -n $CONTRACT | openssl dgst -sha256 | sed 's/.*= //g')
+CONTRACT_HASH=sha256sum | awk '{ print $CONTRACT }'
```

Comment:
Does `$CONTRACT` get expanded? I would expect bash to not expand the $ variable because it's within single quotes here.

Reply:
I think there was more than one mention of openssl elsewhere?

Reply:
look at #161, it supersedes this one